### DISCUSSION TITLE
Problem key:generate goravel v1.13.9

### DISCUSSION THREAD

**Discussion by arifrahmanfauzi** [goravel/goravel#403] [General]
after i clone goravel, when i run *go run . artisan key:generate* i got this error

# modernc.org/sqlite/lib
..\..\pkg\mod\modernc.org\sqlite@v1.23.1\lib\defs.go:13:35: undefined: sqlite3_index_constraint
..\..\pkg\mod\modernc.org\sqlite@v1.23.1\lib\defs.go:14:35: undefined: sqlite3_index_orderby
..\..\pkg\mod\modernc.org\sqlite@v1.23.1\lib\defs.go:15:35: undefined: sqlite3_index_constraint_usage
..\..\pkg\mod\modernc.org\sqlite@v1.23.1\lib\mutex.go:19:5: undefined: Xsqlite3_threadsafe
..\..\pkg\mod\modernc.org\sqlite@v1.23.1\lib\mutex.go:29:11: undefined: Xsqlite3_config
..\..\pkg\mod\modernc.org\sqlite@v1.23.1\lib\mutex.go:29:32: undefined: SQLITE_CONFIG_MUTEX
..\..\pkg\mod\modernc.org\sqlite@v1.23.1\lib\mutex.go:29:122: undefined: SQLITE_OK
..\..\pkg\mod\modernc.org\sqlite@v1.23.1\lib\mutex.go:30:8: undefined: Xsqlite3_errstr
..\..\pkg\mod\modernc.org\sqlite@v1.23.1\lib\mutex.go:40:17: undefined: Sqlite3_mutex_methods
..\..\pkg\mod\modernc.org\sqlite@v1.23.1\lib\mutex.go:239:46: undefined: SQLITE_OK
..\..\pkg\mod\modernc.org\sqlite@v1.23.1\lib\mutex.go:239:46: too many errors
# github.com/bytedance/sonic/internal/rt
..\..\pkg\mod\github.com\bytedance\sonic@v1.10.2\internal\rt\gcwb.go:102:6: missing function body
..\..\pkg\mod\github.com\bytedance\sonic@v1.10.2\internal\rt\gcwb.go:104:6: missing function body
..\..\pkg\mod\github.com\bytedance\sonic@v1.10.2\internal\rt\gcwb.go:114:6: missing function body

i am using goravel v1.13.9

**Comment by hwbrzzl** [comment]
Do you run `go mod tidy` first?

**Reply by arifrahmanfauzi** [reply]
sure i have do `go mod tidy` after clone that

**Reply by hwbrzzl** [reply]
Em, first time I encountered this, which is your Golang version? Do you try to google it?

**Reply by arifrahmanfauzi** [reply]
last i try v1.20

**Reply by arifrahmanfauzi** [reply]
but after i update golang into v1.22.2, all is clear, i didn't see any error again

**Reply by hwbrzzl** [reply]
Em, v1.20 should be ok, there is a test to cover it: https://github.com/goravel/framework/actions/runs/8892009340/job/24415166144

**Reply by arifrahmanfauzi** [reply]
ok, thank you very much
